app/config.json

```diff
@@ -1,22 +1,15 @@
 {
```

Comment:
Please leave this file as it is. It is used to show some default builds with the default theme.